panel_grid.py

```diff
@@ -112,8 +115,10 @@ def labeling(self, tab, i, j, element):
     def add_week(self):
         pass
 
-    def add_activity(self):
-        pass
+    def add_activity(self, activity, table):
+        week = self.t.timeline["week" + str(self.week)]
+        self.t.add_activity(week, activity)
+        self.show_table(self.t.timeline["week" + str(self.week)], table)
```

Comment:
2nd (not really a bug) that needs your immediate attention:
We have now successfully completed the adding an activity, but what if the user typed wrong... (can never trust these users shm...)

Your mission, should you choose to accept it: **Creating a remove button, with a placeholder so that the user can choose what to remove**